usaspending_api/awards/v2/views/idvs/funding_accounts.py

```diff
@@ -111,34 +105,21 @@ def post(self, request: Request) -> Response:
 
 
 @validate_post_request(TREEMAP_MODELS)
-class IDVFundingTreemapViewSet(IDVFundingBaseViewSet):
-    """THIS ONE"""
+class IDVFundingAccountViewSet(IDVFundingBaseViewSet):
 
     @cache_response()
     def post(self, request: Request) -> Response:
-        sql = SQL("")
-        funding_agency_from = SQL("""
-            inner join agency a on
-                a.id = ca.funding_agency_id
-            inner join toptier_agency ta on
-                ta.toptier_agency_id = a.toptier_agency_id
-        """)
         offset = SQL("offset {}".format((request.data['page'] - 1) * request.data['limit']))
         order_by = SQL("order by {} {}".format(SORTABLE_COLUMNS[request.data['sort']], request.data['order']))
-        group_by = SQL("""group by federal_account, fa.account_title, cte.award_id, ca.funding_agency_id,
-                        ta.abbreviation, ta.name, a.id""")
+        group_by = SQL("""group by federal_account, fa.account_title, cte.award_id""")
         columns = SQL("""sum(nullif(faba.transaction_obligated_amount, 'NaN')) total_transaction_obligated_amount,
                       taa.agency_id || '-' || taa.main_account_code federal_account,
                       sum(sum(nullif(faba.transaction_obligated_amount, 'NaN')))
                            over (partition by cte.award_id) as total,
-                      fa.account_title,
-                      ca.funding_agency_id,
-                      ta.abbreviation,
-                      ta.name,
-                      a.id agency_id
+                      fa.account_title
                     """)
 
-        results = self._business_logic(request.data, columns, funding_agency_from, group_by, order_by, offset, sql)
+        results = self._business_logic(request.data, columns, group_by, order_by, offset)
```

Comment:
`get_pagination()` requires the full list of DB rows to function. If a SQL offset is used, the pagination breaks and the last two pages of results aren't returned.